stan/math/prim/err/check_flag_sundials.hpp

```diff
@@ -28,9 +29,8 @@ inline void check_flag_kinsol(int flag,
                               long int max_num_steps) {  // NOLINT(runtime/int)
   std::ostringstream ss;
   if (flag == -6) {
-    ss << "algebra_solver: max number of iterations: " << max_num_steps
```

Comment:
Both files in this file don't have complete doxygen (param and throw are missing). Mind fixing both here? The check_flag_kinsol should be fixed as its being modified.